.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.19.2
+  shared: getoutreach/shared@dev:jaredallard-feat-yarn-v3
```

Comment:
Change back to `dev:first` before merging (publish to `dev:first` before merging locally)